CMSgov/onemac
# OY2 15971: Updated Waiver read-only form: Amendment details

Story: https://qmacbis.atlassian.net/browse/OY2-15971
Endpoint: https://d31ngau906tfle.cloudfront.net/

### Details
Creating the Waiver Amendment Details page

### Changes
- Updated the DetailsView to allow the vertical nav to be optional
- updated the back end query to be able to handle Amendments

### Implementation Notes
- Developer focused info that may affect future work

### Test Plan
1. Login as a user that can see waiver packages
2. Navigate to the Waiver Tab and open a Waiver family
3. Click on a Waiver Amendment
4. View the details and verify that they match the story ACs and the Figma, where appropriate.
